tldw_Server_API/tests/DB_Management/test_media_db_schema_bootstrap.py

```
@@ -0,0 +1,68 @@
+import pytest
+from tldw_Server_API.app.core.DB_Management.Media_DB_v2 import MediaDatabase
+from tldw_Server_API.app.core.DB_Management.backends.base import BackendType
+from tldw_Server_API.app.core.DB_Management.media_db.schema.bootstrap import ensure_media_schema
+from tldw_Server_API.app.core.DB_Management.media_db.schema import bootstrap as bootstrap_module
+
+
+def test_ensure_media_schema_dispatches_sqlite(monkeypatch) -> None:
+    db = SimpleNamespace(backend_type=BackendType.SQLITE)
+    calls: list[object] = []
+
+    monkeypatch.setattr(bootstrap_module, "initialize_sqlite_schema", lambda value: calls.append(value))
```

Ruff (0.15.6)

[warning] Lambda may be unnecessary; consider inlining inner function

Inline function call

(PLW0108)